mlrun/api/db/sqldb/db.py

```diff
@@ -3947,6 +3939,12 @@ def set_run_notifications(
             )
         self._commit(session, [run], ignore=True)
 
+    @staticmethod
+    def _transform_datastore_profile_model_to_schema(
+        query,
```

Comment:
```suggestion
        db_object,
```
Or `db_record`. You don't get a query here.